makefiles/arch/riscv.inc.mk

```diff
@@ -4,8 +4,8 @@ export TARGET_ARCH ?= riscv-none-embed
 # define build specific options
 CFLAGS_CPU   = -march=rv32imac -mabi=ilp32 -mcmodel=medlow -msmall-data-limit=8
 CFLAGS_LINK  = -nostartfiles -ffunction-sections -fdata-sections
-CFLAGS_DBG  ?= -g3 -Og
-#CFLAGS_OPT  ?= -Os
+CFLAGS_DBG  ?= -ggdb -g3
```

Comment:
Shouldn't we rather strife for having the potential bug pointed out by the `riscv` toolchain fixed in `wolfssl` instead of hiding it for `riscv`? Until then we can deactivate `-Werror=maybe-uninitialized` for _`wolfssl`_.

Reply:
In fact, this commit is also fixing the issue that Murdock is not building with the same optimization flag than a local build.